### DISCUSSION TITLE
Does rapids support CBO optimization ?[QST]

### DISCUSSION THREAD

**Discussion by chenrui17** [NVIDIA/spark-rapids#5397] [General] [answered]
like cpu spark , we can set spark.sql.cbo.enabled true ,and spark.sql.cbo.joinReorder.enabled true to Adjust the order of the join.if i set this config on rapids, can it work ?

**Comment by revans2** [comment, marked as answer]
The rapids accelerator operates at the level of the physical plan.  The CBO works at the level of the logical plan so it should work out of the box.  We have not tested this yet.  If you run into issues please let us know.

**Comment by sameerz** [comment]
@chenrui17 if you have any further questions about this please reopen or file another issue.
